### PR TITLE
Test the pip-installed CLI

### PR DESCRIPTION
**Pull request by xbrianh**
This tests the CLI as it is installed into a Python virtual
environment.

Note that it is installed in dev_scripts and executed externally
from the standard unit tests suite. Coverage and other utilities
are not meaningful for this tests, and interfere with output.

### CODE REVIEW

File: dev_scripts/test_installed_cli.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+            workflow_id = d['workflows'][0]['workflowId']
+
+        with self.subTest("get-workflow"):
+            d = run_json(f"{TNU} workflows get-workflow {WORKSPACE_ARGS}"
+                         f" --submission-id {submission_id} --workflow-id {workflow_id}")
+
+        with self.subTest("get-submission"):
+            run(f"{TNU} workflows estimate-submission-cost {WORKSPACE_ARGS} --submission-id {submission_id}")
+
+class bcolors:
```
Comment (DailyDreaming) [inline dev_scripts/test_installed_cli.py:170]: `Bcolors`

File: dev_scripts/test_installed_cli.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+def run(cmd: str) -> subprocess.CompletedProcess:
+    """This is meant intended mimic typical CLI usage, so execute in a shell."""
+    p = subprocess.run(cmd, shell=True, capture_output=True)
+    try:
+        p.check_returncode()
+        print(f"{bcolors.OKGREEN}{cmd}{bcolors.ENDC}")
+    except subprocess.CalledProcessError:
+        print(f"{bcolors.FAIL}{cmd}{bcolors.ENDC}")
+        print(f"{bcolors.FAIL}{p.stderr}{bcolors.ENDC}")  # type: ignore # go home mypy you're drunk
```
Comment (DailyDreaming) [inline dev_scripts/test_installed_cli.py:189]: lol, this should be the de facto comment for mypy.

**Review by DailyDreaming — COMMENTED** [review]
LGTM.

**Review by DailyDreaming — APPROVED** [review]

File: dev_scripts/test_installed_cli.py (hunk 1 above)
Comment (xbrianh) [reply dev_scripts/test_installed_cli.py:170]: thanks, will fix!

**Pull request merged**